google/cloud/internal/rest_opentelemetry_test.cc

```diff
@@ -55,16 +55,17 @@ TEST(RestOpentelemetry, MakeSpanHttp) {
 
   auto spans = span_catcher->GetSpans();
   EXPECT_THAT(
-      spans, ElementsAre(AllOf(
-                 SpanHasInstrumentationScope(), SpanKindIsClient(),
-                 SpanNamed("HTTP/GET"),
-                 SpanHasAttributes(
-                     SpanAttribute<std::string>(sc::kNetTransport,
-                                                sc::NetTransportValues::kIpTcp),
-                     SpanAttribute<std::string>(sc::kHttpMethod, "GET"),
-                     SpanAttribute<std::string>(sc::kHttpUrl, kUrl),
-                     SpanAttribute<std::string>("http.header.authorization",
-                                                secret.substr(0, 32))))));
+      spans,
+      ElementsAre(AllOf(
+          SpanHasInstrumentationScope(), SpanKindIsClient(),
+          SpanNamed("HTTP/GET"),
+          SpanHasAttributes(
+              SpanAttribute<std::string>(sc::kNetTransport,
+                                         sc::NetTransportValues::kIpTcp),
+              SpanAttribute<std::string>(sc::kHttpMethod, "GET"),
+              SpanAttribute<std::string>(sc::kHttpUrl, kUrl),
+              SpanAttribute<std::string>("http.request.header.authorization",
+                                         secret.substr(0, 32))))));
```

Comment:
D'oh, forgot to add this to the approval review. Sorry!

We should probably test all three branches

- `http.request.header`
- `http.request.header.authorization`
- `http.request.header.other`

Reply:
Done. Except that I do not think `http.request.header` can be a thing, you probably meant an empty *value*.

Reply:
Yes, I realized and edited my comment within 1 minute of posting it.... It was already too late. :(